packages/bsky/src/api/blob-resolver.ts

```diff
@@ -30,7 +31,24 @@ export const createRouter = (ctx: AppContext): express.Router => {
         return next(createError(400, 'Invalid cid'))
       }
 
-      const { pds } = await ctx.didResolver.resolveAtpData(did) // @TODO cache did info
+      const [{ pds }, takedown] = await Promise.all([
+        ctx.didResolver.resolveAtpData(did), // @TODO cache did info
+        ctx.db.db
+          .selectFrom('moderation_action_subject_blob')
```

Comment:
do you need a `.select()` or `selectAll()` here?

Reply:
I double-checked and it does work without it, but good call!  Definitely should have one.